apps/new-tab/src/Settings.tsx

```
@@ -111,6 +142,17 @@
       }
     });
   });
+  function removePlugin(plugin: PluginItem) {
+    const parsedPlugins =
+      typeof activePlugins() == "object"
+        ? activePlugins()
+        : JSON.parse(activePlugins().toString());
+    setActivePlugins(
+      parsedPlugins.filter(
+        (plugin: PluginItem) => plugin.fileName !== plugin.fileName
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)